plugin/src/lib.rs

```diff
@@ -294,17 +370,42 @@ pub extern "C" fn main() {
         online::install();
         skyline::patching::Patch::in_text(0x14f97bc).nop().unwrap();
         skyline::patching::Patch::in_text(0x1509dc4).nop().unwrap();
-        skyline::install_hooks!(training_reset_music1, training_reset_music2, main_menu_quick, title_screen_play, sss_to_css, css_to_sss);
+        skyline::install_hooks!(
+            training_reset_music1,
+            training_reset_music2,
+            main_menu_quick,
+            title_screen_play,
+            sss_to_css,
+            css_to_sss,
+            copy_fighter_info,
+            load_ingame_call_sequence_scene,
+            load_melee_scene,
+            game_end,
+            game_exit
+        );
     }
 
     #[cfg(not(feature = "runtime"))]
-    { utils::init(); }
+    {
```

Comment:
Seems to do it on one liner functions, but whatever (Does Rust support no curly brace on one line functions?)
`#[cfg(not(feature = "runtime"))]
          utils::init();`
or something?

Reply:
don't think so bad